storage/diskmetricstore.go

```diff
@@ -100,6 +100,17 @@ func (dms *DiskMetricStore) SubmitWriteRequest(req WriteRequest) {
 	dms.writeQueue <- req
 }
 
+// Wipe implements the MetricStore interface.
+func (dms *DiskMetricStore) Wipe() {
+	// Delete all metric groups by sending write requests with MetricFamilies equal to nil
+	for _, group := range dms.GetMetricFamiliesMap() {
+		dms.SubmitWriteRequest(WriteRequest{
+			Labels:    group.Labels,
+			Timestamp: time.Now(),
+		})
```

Comment:
This implementation is so nice and elegant and it only uses other methods from the interface (rather than any implementation details) of the MetricStore. Thus, I have second thoughts now if we really want a `Wipe` method in the interface. This code could as well live in `wipe.go`. What do you think?

Reply:
It's a good point, my two cents follow

Since it's possible to wipe the entire MetricStore (no matter whether this is the Disk implementation or any other future MetricStore implementation) by simply using already existing methods from the interface, we could avoid forcing future MetricStore implementations to implement their own Wipe() method, which would most likely be just the same code.

Keeping Wipe() method within the interface would make sense having plans to call it from within the MetricStore, it'd be nice if we could somehow and "elegantly" mixin a generic Wipe implementation method without forcing to each implementation to create its own and having it available through a pointer receiver method, but I'm not sure if this is possible.

Reply:
The latter could still be addressed by refactoring the code into a helper function, should the need ever come up.

Reply:
OK, let's move the wipe logic to the handler then